eva-lib/src/test/resources/eva.properties

```diff
@@ -7,6 +7,7 @@ eva.mongo.read-preference=
 
 eva.mongo.collections.variants=vars
 eva.mongo.collections.files=files
+eva.mongo.collections.annotation_metadata=annotation_metadata
```

Comment:
@cyenyxe and I spoke about this and I have currently changed it in another eva-ws PR (https://github.com/tomdcsmith/eva-ws/blob/479a2d2f60a900b5457740c29324b72f51ec299b/eva-lib/src/test/resources/application.properties) to "db.collections.annotation-metadata.name" for the reason you mention. I think I agree with possibly removing ".name" from the property names.